### ISSUE TITLE
HTML_CodeSniffer with PhantomJS doesn't work.

### ISSUE THREAD

**Issue by EugenMazanik** [squizlabs/HTML_CodeSniffer#215]
> phantomjs HTMLCS_Run.js http://example.com WCAG2A
Have error:
"ReferenceError: Can't find variable: HTMLCS_RUNNER"
in 
[page.evaluate(function(standard) {HTMLCS_RUNNER.run(standard);}, standard);](https://github.com/squizlabs/HTML_CodeSniffer/blob/aec6f3718afb3a4fa1c410567529276a1167ebfd/Contrib/PhantomJS/HTMLCS_Run.js#L170)

Grunt does not need.
> If you are using command-line processing, you don't need to build the auditor as above.

**Comment by ironikart** [comment]
I've updated the instructions in the README, they were outdated. You will need a built version exported to `./build/HTMLCS.js` (the default location using Grunt).

We use PhantomJS for the unit tests, you can check the `Contrib/Testing/` folder for an example.

**Comment by EugenMazanik** [comment]
Could you please take a look in log:

[DEBUG] WebPage - updateLoadingProgress: 90
[DEBUG] WebPage - updateLoadingProgress: 100
[DEBUG] WebPage - setupFrame ""
[WARNING] Can't open 'undefined/../../build/HTMLCS.js'
[DEBUG] WebPage - evaluateJavaScript "(function() { return (function (standard) {HTMLCS_RUNNER.run(standard);})(\"WCAG2AA\"); })()"
ReferenceError: Can't find variable: HTMLCS_RUNNER

Probably error here: https://github.com/squizlabs/HTML_CodeSniffer/blob/aec6f3718afb3a4fa1c410567529276a1167ebfd/Contrib/PhantomJS/HTMLCS_Run.js#L161

Locally I changed from
`page.injectJs(cwd + '/../../build/HTMLCS.js');`
to (like in Contrib/Testing/HTMLCS_Test.js)
`page.injectJs('../../build/HTMLCS.js');`

It helps.

**Comment by ironikart** [comment]
It looks like the `cwd` variable which may not be available on the system you are running this from. This is using [`system.env['PWD']`](http://phantomjs.org/api/system/property/env.html)

Can you please post some details about the following:
- Your operating system version.
- Your version of PhantomJS.

**Comment by rakeshkmr516** [comment]
KInd of the same issue I got using geckodriver( firefox) of Selenium throwing TypeError: window.HTMLCS_RUNNER is undefined
Issue Link: https://github.com/squizlabs/HTML_CodeSniffer/issues/219

Interesting thing is am able to get the same code work using chromedriver of selenium runs code on Chrome browser

**Comment by EugenMazanik** [comment]
OS: Windows 10 Pro
PhantomJS 2.1.1

**Comment by ironikart** [comment]
Reference issue #227
